src/rss.py

```diff
@@ -8,39 +8,30 @@
 if hasattr(ssl, '_create_unverified_context'):
     ssl._create_default_https_context = ssl._create_unverified_context
 
-# Création d'une instance
 news_feed = feedparser.parse('https://feed.ausha.co/B6Ad3sAApEzX')
 
-def tags_list_flux_rss():
-    """Retourne les tags du podcast se trouvant le dossier local "Podcast" """
-    for x in Path('Podcast').iterdir():
-        s = x.name
-    # Propriétés du flux
-    tags = []
-    for i in news_feed.entries:
-        if i['title'] in s:
-            pprint(i['title'])
-            for z in i['tags']:
-                if z['term'] is not None:
-                    tags.append(z['term'])
-            break
-    return tags
-
-
-def description_flux_rss():
-    """Retourne les tags du podcast se trouvant le dossier local "Podcast" """
-    for x in Path('Podcast').iterdir():
-        s = x.name
-    # Propriétés du flux
-    for i in news_feed.entries:
-        if i['title'] in s:
-            description = i['content'][0]['value']
+def get_description_from_rss(podcast_title):
+    for meditation_podcast in news_feed.entries:
```

Comment:
Attention au nommage de tes variables de boucle, mieux vaut écrire `for entry in news_feed.entries`